src/components/InputField/InputField.css

```diff
@@ -1,5 +1,13 @@
+@import "../../GlobalStyles.css";
+
+:root {
+  --textfield-background: #fff;
```

Comment:
you already have this hex as a variable in GlobalStyles. `primary-btn-text`. Also this is a right time to rename `primary-btn-text` to something more generic like `light`. PS, since this is a custom component, you don't have to populate `root`. You should be able to use variables without root